app/players/elves/elf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Elf(Player):
```

Comment:
The `Elf` class does not implement all abstract methods of `Player`, so it should be abstract as well. Inherit `Elf` from `ABC` as well.